docs/rules/named-import.md

```diff
@@ -0,0 +1,83 @@
+# Prevent named imports from react (react/named-import)
+
+Enforce the use of named imports from React, so it's visible in code that it's React and not custom code. This rule also creates consistency inside a codebase because it enforces or prohibits the use of specified named imports.
```

Comment:
```suggestion
Prohibit the use of named imports from React, so it's visible in code that it's React and not custom code; or enforce the use of named imports from React, to avoid needing the `React` default import. This rule also creates consistency inside a codebase because it enforces or prohibits the use of specified named imports.
```